server/export.js

```diff
@@ -80,6 +80,11 @@ export default async function (dir, options) {
     hotReloader: null
   }
 
+  // Static hosted sites sometimes have different url requirements
+  if (config.exportAssetPrefix) {
+    renderOpts.assetPrefix = config.exportAssetPrefix.replace(/\/$/, '')
```

Comment:
Because the folder structure and asset path are different on the exported site versus local. Example: github pages project site where it's at organization.github.io/project-name but the files actually live in the root of that project repo. Using just assetPrefix breaks local development